syntax = "proto3";

import "common.proto";

// root message that captures all the data
message Info {
  repeated FlightData data = 3;
  common.common.Seat seat = 9;
  repeated common.common.Passenger passengers = 8;
  common.common.Trip trip = 19;
}

message FlightData {
  string date = 2;
  common.common.Airport from_flight = 13;
  common.common.Airport to_flight = 14;
  optional int32 max_stops = 5;
}